proto/tendermint/types/types.proto

```diff
@@ -227,3 +228,22 @@ message MalleatedTx {
   bytes original_tx_hash = 1;
   bytes tx               = 2;
 }
+
+// Proof represents proof of a namespace.ID in an NMT.
+// In case this proof proves the absence of a namespace.ID
+// in a tree it also contains the leaf hashes of the range
+// where that namespace would be.
+message NMTProof {
+  // start index of this proof.
+  int32 start = 1;
+  // end index of this proof.
+  int32 end = 2;
+  // Nodes that together with the corresponding leaf values
+  // can be used to recompute the root and verify this proof.
+  repeated bytes nodes = 3;
```

Comment:
In the comment, can you describe the expected size of each `bytes`?

Reply:
good idea
https://github.com/celestiaorg/celestia-core/pull/615/commits/11ba904fff5a75afebf96e5bcf2b1b3c78249053